Cargo.toml

```diff
@@ -19,7 +19,7 @@ strict_gc = []
 
 [dependencies]
 arrayvec = { version = "0.4", default-features = false }
-crossbeam-utils = { version = "0.1", default-features = false }
+crossbeam-utils = { version = "0.2", default-features = false }
```

Comment:
I think we need `crossbeam-utils` 0.2.1, right?

Reply:
I normally only put the major version in Cargo.toml. Cargo will automatically use the latest compatible version (0.2.1, 0.2.2, etc).

Reply:
I'm aware that specifying version = "0.2" works and it is an accepted convention to ignore the patch version in specifying dependency.  However, in my understanding, it's an undocumented behavior of `cargo`, and pedantically speaking, a future version of `cargo` may change its dependency resolution algorithm.

For this reason, I prefer to specify version = "0.2.1".

But I'm also aware that this matter is really minor: it's highly likely that specifying version = "0.2" will work correctly forever.  Since we're in front of a big 0.1.0 release, it's more beneficial to merge this PR now.  I'm approving and merging this PR.